vector-config/src/main/res/values/urls.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- This file contains url values-->
+
+    <string name="threads_learn_more_url" translatable="false">https://element.io/help#threads</string>
```

Comment:
This anchor `threads` does not exist (yet), and today this page does not mention thread the user experience will be quite bad. But @ariskotsomitopoulos told me that the webpage will be ready today or tomorrow.
So OK to merge the PR.

Reply:
Yeap, it should be deployed before the release as we discussed. All platforms will release that url

Reply:
Seems to be live now 👍

Reply:
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/3940906/161826455-692f41b9-4be1-44f3-92ea-165498b04cb8.png">